### ISSUE TITLE
ComputeDeltas equation is not rendered

### ISSUE THREAD

**Issue by keunwoochoi** [pytorch/audio#638]
https://pytorch.org/audio/functional.html#compute-deltas

A closing parenthesis `}` is missing, but I'm not entirely sure what it is supposed to be so instead of making a PR I'm reporting this.

**Comment by mthrok** [comment]
Thanks for reporting. I will take a look at it.

**Comment by vincentqb** [comment]
The bracket is missing for the upper bound in the final sum :) Feel free to open a PR for this!
```
\[d_t = \frac{\sum_{n=1}^{\text{N}} n (c_{t+n} - c_{t-n})}{2 \sum_{n=1}^{\text{N}} n^2} \]
```

**Comment by mthrok** [comment]
It's unbalanced curly bracket. Will be fixed in #788.

**Comment by mthrok** [comment]
Fixed by #788 

https://pytorch.org/audio/0.7/functional.html#compute-deltas
